### PR TITLE
Add Auxilium crypto support

### PR DESCRIPTION
**Pull request by auxiliumglobal**
Auxilium (AUX) is a cryptocurrency foundation of a philantropic company Auxilium Global. Its current main network started in summer of 2018, after performing swap from Sprouts blockchain and running a modified fork of Ethereum Go implementation.

- Blockchain explorer: https://explore.auxilium.global
- R-RPC node: https://rpc.auxilium.global
- Node: https://github.com/auxiliumglobal/go-ethereum
- ChainID: 28945486
- SLIP-44 number: 344 

More information about Auxilium Global can be found here: https://auxilium.global

Gracias!

### CODE REVIEW

**Comment by CryptoQueen628** [comment]
Thanks, we will look into this!

File: common/features/config/networks/static/reducer.ts [hunk 1]
```diff
@@ -999,6 +1000,31 @@ export const STATIC_NETWORKS_INITIAL_STATE: types.ConfigStaticNetworksState = {
...
+    isCustom: false,
+    color: '#85dc35',
+    blockExplorer: makeExplorer({
+      name: 'Auxilium Explore',
+      origin: 'https://explore.auxilium.global/'
+    }),
+    tokens: [],
+    contracts: [],
+    dPathFormats: {
+      [SecureWalletName.TREZOR]: AUX_DEFAULT,
```
Comment (mkrufky) [inline common/features/config/networks/static/reducer.ts]: Please do not advertise Trezor support without actually supporting the Trezor hardware wallet.  You should remove this line for now - you may add it back in a separate pull request later on if Trezor releases support for your chain.

File: common/features/config/networks/static/reducer.ts [hunk 2]
```diff
@@ -999,6 +1000,31 @@ export const STATIC_NETWORKS_INITIAL_STATE: types.ConfigStaticNetworksState = {
...
+      [SecureWalletName.SAFE_T]: AUX_DEFAULT,
+      [SecureWalletName.LEDGER_NANO_S]: AUX_DEFAULT,
```
Comment (mkrufky) [inline common/features/config/networks/static/reducer.ts]: Please do not advertise Ledger support without actually supporting the Ledger hardware wallets.  You should remove this line for now - you may add it back in a separate pull request later on if Ledger releases support for your chain.

**Comment by SharonManrique** [comment]
@mkrufky Thank you for review this for us. We highly appreciate the help!

**Comment by SharonManrique** [comment]
@auxiliumglobal Thank you for you contribution! It has been tagged for Release 1.7.0 :)

**Pull request merged**